daemon/cmd/daemon_main.go

```diff
@@ -1047,6 +1047,9 @@ func initializeFlags() {
 	flags.Bool(option.EnableK8sTerminatingEndpoint, true, "Enable auto-detect of terminating endpoint condition")
 	option.BindEnv(option.EnableK8sTerminatingEndpoint)
 
+	flags.Bool(option.DisableSipVerification, defaults.DisableSipVerification, "Disable source ip verification")
+	option.BindEnv(option.DisableSipVerification)
```

Comment:
Please name this `EnableSipVerification` and set the default to `true` and reverse the logic internally. Otherwise we often end up in a confusing discussion about what it means to disable the option to disableSipVerification, because double-negatives have different meaning in different languages.
```suggestion
	flags.Bool(option.EnableeSipVerification, defaults.EnableSipVerification, "Enable source ip verification")
	option.BindEnv(option.EnableSipVerification)
```